packages/example/pages/_app.tsx

```diff
@@ -246,9 +254,36 @@ function RainbowKitApp({
 
   const [isMounted, setIsMounted] = useState(false);
   useEffect(() => setIsMounted(true), []);
+  // biome-ignore lint/nursery/useExhaustiveDependencies: TODO
+  useEffect(() => {
+    wagmiConfig.setConnectors(connectors);
+  }, [locale]);
```

Comment:
✨ In the singleton approach with `i18n`, connectors run only once and aren’t updated thereafter. For every page load or locale change, we instruct wagmi to reset the connectors, enabling `connectorsForWallets` to reflect the latest `i18n` singleton class locale 🙌. 

Through `useMemo` from `I18nProvider` will update singleton locale from `i18n` throughout the app's runtime. https://github.com/rainbow-me/rainbowkit/pull/1534/files#r1341737308

Reply:
I tried to do this force refresh connector logic in `RainbowKitProvider` by using `useConfig` hook and `setConnector` function from wagmi, but it crashes and doesn't seem to be ready yet.